### PR TITLE
Fixes #14856: Changing visual seek behaviour in Vv

### LINKED ISSUE #14856: `Vv` + `g` (seek offset) doesn't work properly
`r2 /bin/ls` then `aaa`, then go into `Vv` mode. It shows us `entry0` function. If we call the `seek to offset` command (`g` key), and type `main`, it doesn't jump to main and shows it. Note, that `main` is not in the visible part of the functions list. If we seek into any function that is visible in the functions list window, e.g. `sym.imp.strrchr`, it will show `>` character before it, but will not show the function body in the corresponding window.

### PR DESCRIPTION
**Pull request by deepakchethan**

### CODE REVIEW

File: libr/core/vmenus.c [hunk 1]
```diff
@@ -3361,10 +3361,20 @@ R_API void r_core_visual_anal(RCore *core, const char *input) {
 				}
 			}
 			break;
-		case 'g':
-		        r_core_visual_showcursor (core, true);
-		        r_core_visual_offset (core);
-		        r_core_visual_showcursor (core, false);
+		case 'g': {
+			r_core_visual_showcursor(core, true);
```
Comment (radare) [inline libr/core/vmenus.c]: missing space before (

File: libr/core/vmenus.c [hunk 2]
```diff
@@ -3361,10 +3361,20 @@ R_API void r_core_visual_anal(RCore *core, const char *input) {
...
+			r_core_visual_offset(core);        // change the seek to selected offset
+			RListIter *iter;		   // change the curren option to selected seek
+			RAnalFunction *fcn;
+			int i = 0;
+			r_list_foreach (core->anal->fcns, iter, fcn) {
+				if (core->offset ==  fcn->addr){
```
Comment (radare) [inline libr/core/vmenus.c]: missing space ){

File: libr/core/vmenus.c (hunk 1 above)
Comment (deepakchethan) [reply libr/core/vmenus.c]: Added

File: libr/core/vmenus.c (hunk 2 above)
Comment (deepakchethan) [reply libr/core/vmenus.c]: Added

**Pull request merged**